### DISCUSSION TITLE
Reports

### DISCUSSION THREAD

**Discussion by ghost** [postalserver/postal#2000] [Help with using Postal]
Hi to all colegues on Postal Community. I have a question (not a problem), but first, thanks in advance for anyone who can help me.

After a few days studing the documentation and all posts on installation category, i started my environment and send my first Mautic campaign with sucess. Everything work according my plan. Of course i had a few troubles but as i said, this community was very important to fix all of them.

Now, here my question;

I need one "report" also known as sql query to know all mails with determinet condition (status: hardfail). I tryed to extract from web but in my case i have 309 pages with 30 results each one, so, its impossible.

Anyone know some way to get this information?

**Comment by willpower232** [comment]
You should be able to find this information in the database, I couldn't tell you the exact query off the top of my head but it is worth saying that the information won't stay in Postal forever so if you need that information long term then you will need something to listen to the webhooks or something.
